### DISCUSSION TITLE
quill Cannot import modules/Mention. Are you sure it was registered?

### DISCUSSION THREAD

**Discussion by aftabkhaliq** [afconsult/quill-mention#308] [Q&A] [answered]
i am having issue  register quill mention with  quill  in webpack/mix as I am using it with laravel 

here is my quill.js file 

**import Quill from 'quill/core';
import mention from 'quill-mention';
Quill.register('modules/mention', mention,true);
export default Quill;**

works fine without mention  but as soon as I use mention it throw error
app.layout
            <script>
                const
                atValues = [
                    {id: 1, value: "Fredrik Sundqvist"},
                    {id: 2, value: "Patrik Sjölin"}
                ];
                const hashValues = [
                    {id: 3, value: "Fredrik Sundqvist 2"},
                    {id: 4, value: "Patrik Sjölin 2"}
                ];

                var quill = new Quill("#quilEditor", {
                    debug: "info",
                    theme: "snow",
                    modules: {
                        Mention: {
                            allowedChars: /^[A-Za-z\sÅÄÖåäö]*$/,
                            mentionDenotationChars: ["@", "#"],
                            source: function (searchTerm, renderList, mentionChar) {
                                let values;

                                if (mentionChar === "@") {
                                    values = atValues;
                                } else {
                                    values = hashValues;
                                }

                                if (searchTerm.length === 0) {
                                    renderList(values, searchTerm);
                                } else {
                                    const matches = [];
                                    for (i = 0; i < values.length; i++)
                                        if (
                                            ~values[i].value.toLowerCase().indexOf(searchTerm.toLowerCase())
                                        )
                                            matches.push(values[i]);
                                    renderList(matches, searchTerm);
                                }
                            }
                        }
                    }
                });

            </script>

![image](https://user-images.githubusercontent.com/78808706/148343508-8551b902-2a84-4ddc-9c76-175348c8e456.png)

**Comment by fahrenkrug** [comment, marked as answer]
Did you tried to lowercase the `Mention` in the `modules` ?

```
modules: {
                    Mention: {
```

**Comment by avinash-threado** [comment]
Add this dependency: `"quill-mention": "4.1.0"`
